nethserver-roundcubemail-next.spec

```diff
@@ -15,8 +15,7 @@ BuildArch: noarch
 
 BuildRequires: nethserver-devtools
 Requires: nethserver-httpd, nethserver-mail-server
-Conflicts: nethserver-roundcubemail
```

Comment:
Can we still ask to conflict `nethserver-roundcubemail` we need to be sure it has been removed